includes/site-health/load.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Site Health tests loader.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! has_filter( 'user_has_cap', 'wp_maybe_grant_site_health_caps' ) ) {
+	return;
+}
+
+// Site health tests to load.
+$site_health_tests = array(
+	'audit-autoloaded-options',
+	'audit-enqueued-assets',
+	'webp-support'
+);
+
+foreach ( $site_health_tests as $test ) {
+	require_once __DIR__ . '/' . $test . '/helper.php';
+	require_once __DIR__ . '/' . $test . '/hooks.php';
+}
```

Comment:
I think this is a bit "too smart".

We don't want to mandate which files a Site Health check should have. For that reason, I think it's better to make this a simple list of the 6 `require_once`s necessary, so that _any_ new file can be simply added to that list.

Also there's a PHPCS lint error in the array, but that shouldn't be relevant anymore if you apply my suggested changes.

Reply:
I'm also thinking the same. Thanks for pointing in the right direction.